way_to_home/static/src/components/loginForm/loginForm.js

```diff
@@ -156,17 +168,20 @@ class LoginForm extends Component {
                             size='medium'
                             disabled={this.state.disable_button}
                             onClick={this.onClickConfirm}>
-                        {this.state.request_type}
+                        {this.state.confirm_button}
                     </Button>
                     <Button
                         variant='contained'
                         color='secondary'
                         size='medium'
                         className='Btn'
                         onClick={this.props.close}>
-                        cancel
+                        відмінити
                     </Button>
                 </div>
+                {this.state.ajax_error && <CustomizedSnackbars
```

Comment:
Its better to use constant 
after render() add         const { error } = this.state;